Code/Framework/AzToolsFramework/Tests/AssetFileInfoListComparison.cpp

```diff
@@ -59,9 +60,9 @@ namespace UnitTest
             AZ::IO::FileIOBase::SetInstance(nullptr);
             AZ::IO::FileIOBase::SetInstance(m_localFileIO);
 
-            AZ::IO::FileIOBase::GetInstance()->SetAlias("@assets@", m_tempDir.GetDirectory());
+            SetAssetFolder(m_tempDir.GetDirectory());
 
-            AZStd::string assetRoot = AzToolsFramework::PlatformAddressedAssetCatalog::GetAssetRootForPlatform(AzFramework::PlatformId::PC);
+            const AZStd::string assetRoot = AzToolsFramework::PlatformAddressedAssetCatalog::GetAssetRootForPlatform(AzFramework::PlatformId::PC);
```

Comment:
does this test still work in Linux? 
or does PC not matter since it was built out for PC?

Reply:
I don't have a linux environment so I'm keeping an eye on the Jenkins AR linux run, it should run the test.

Reply:
Linux tests passed in Jenkins AR: https://jenkins.build.o3de.org/blue/organizations/jenkins/O3DE/detail/PR-2887/5/pipeline/